src/db/ClimbSchema.ts

```diff
@@ -59,6 +59,28 @@ const MetadataSchema = new Schema<IClimbMetadata>({
   }
 }, { _id: false })
 
+const PitchSchema = new mongoose.Schema({
+  _id: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    default: () => muuid.v4()
+  },
+  uuid: {
+    type: 'string',
+    default: function () { return this._id.toString() }
+  },
+  parentId: { type: String, required: true },
```

Comment:
This implementation allows for *ticking* variations (since each pitch gets a UUID), but not for having multiple parent `climb`s for one `pitch`. I think it is preferable to leave it out because it would reduce clarity a lot in return for what I believe is an unfrequent requirement. Let me explain:

The implementation would require a complimentary field to track the pitches' sequence by `pitch.id`, e. g. `Climb.pitchSequence`: [$id1, $id2, $id3, $id4, ...]. This would either lie outside the `Pitch` object, or make the `Pitch` object nested (e. g. `Pitch.pitchSequence`and `Pitch.data.x`.

Granted, it would indeed be useful for climbing areas with limited space or for mixed climbing (trad/sport, mixed (ice+drytooling)). But I feel like duplicating a pitch here and then seems more feasible. This is based on my ballpark estimate that variations only occur for 5-10% of multi-pitch routes.

Eager to hear your thoughts on this :-)